src/server/api/routers/webhook.ts

```diff
@@ -20,30 +22,35 @@ export const webhookRouter = createTRPCRouter({
         name: 'web',
         active: true,
         events: [
-          'push',
           "issues",
```

Comment:
let's be consistent with single vs double quotes

Reply:
Yeah that does look a bit funky. Will have this fixed on my end 👍🏻